spatialos-sdk/examples/project-example/lib/opt.rs

```diff
@@ -0,0 +1,38 @@
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "project-example",
+    about = "A SpatialOS worker written in Rust."
+)]
+pub struct Opt {
+    #[structopt(name = "WORKER_TYPE", long = "worker-type", short = "w")]
+    pub worker_type: String,
+
+    #[structopt(name = "WORKER_ID", long = "worker-id", short = "i")]
+    pub worker_id: Option<String>,
+
+    #[structopt(name = "POLLING_CONNECTION", long = "polling-connection", short = "p")]
+    pub connect_with_poll: bool,
+
+    #[structopt(subcommand)]
+    pub command: Command,
+}
+
+#[derive(Debug, StructOpt)]
+pub enum Command {
+    #[structopt(name = "receptionist")]
+    Receptionist {
+        host: Option<String>,
```

Comment:
Any reason these are named parameters? Seems inconsistent with the rest of the CLI?

Reply:
I think I had originally though they would be required arguments before I noticed there are default values. Since they're optional, I agree they make more sense as flags 👍